fluent/src/commonMain/kotlin/com/konyaco/fluent/component/Dialog.kt

```diff
@@ -64,63 +82,170 @@ fun Dialog(
-                                        Text(confirmButtonText)
-                                    }
-                                    Button(modifier = Modifier.weight(1f), onClick = onCancel) {
-                                        Text(cancelButtonText)
-                                    }
-                                }
-                            }
-                        }
+                        content = content
+                    )
+                }
+            }
+        }
+    }
+}
+
+enum class ContentDialogButton {
+    Primary, Secondary, Close
+}
+
+@Composable
+fun ContentDialog(
+    title: String,
+    visible: Boolean,
+    content: @Composable () -> Unit,
+    primaryButtonText: String,
+    secondaryButtonText: String? = null,
+    closeButtonText: String? = null,
+    onButtonClick: (ContentDialogButton) -> Unit,
+    size: DialogSize = DialogSize.Standard
+) {
+    FluentDialog(visible, size) {
+        Column {
+            Column(Modifier.background(FluentTheme.colors.background.layer.alt).padding(24.dp)) {
+                Text(
+                    style = FluentTheme.typography.subtitle,
+                    text = title,
+//                  color = FluentTheme.colors.text.text.primary
+                )
+                Spacer(Modifier.height(12.dp))
+                CompositionLocalProvider(
+                    LocalTextStyle provides FluentTheme.typography.body,
+                    LocalContentColor provides FluentTheme.colors.text.text.primary
+                ) {
+                    content()
+                }
+            }
+            // Divider
+            Box(Modifier.height(1.dp).background(FluentTheme.colors.stroke.surface.default))
+            // Button Grid
+            Box(Modifier.height(80.dp).padding(horizontal = 25.dp), Alignment.CenterEnd) {
+                Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+                    AccentButton(modifier = Modifier.weight(1f), onClick = {
+                        onButtonClick(ContentDialogButton.Primary)
+                    }) {
+                        Text(primaryButtonText)
+                    }
+                    if (secondaryButtonText != null) Button(
+                        modifier = Modifier.weight(1f),
+                        onClick = {
+                            onButtonClick(ContentDialogButton.Secondary)
+                        }) {
+                        Text(secondaryButtonText)
+                    }
+                    if (closeButtonText != null) Button(modifier = Modifier.weight(1f), onClick = {
+                        onButtonClick(ContentDialogButton.Close)
+                    }) {
+                        Text(closeButtonText)
                     }
                 }
             }
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+fun ContentDialogHost(state: ContentDialogHostState) {
+    val data = state.currentData
+
+    if (data != null) {
```

Comment:
Try to use a `DialogData.Empy` or Elvis operator﻿ to fix the issue of exit animation terminated